src/internal/clusterstate/v2.0.0.go

```diff
@@ -123,3 +124,76 @@ func SetupPostgresCommitStoreV0(ctx context.Context, tx *pachsql.Tx) error {
 	`)
 	return errors.EnsureStack(err)
 }
+
+var ReposTypeIndex = &col.Index{
```

Comment:
Are any of these functions and types ever referenced externally, should we make them private?